META.coq.in

```diff
@@ -561,4 +561,19 @@ package "plugins" (
     plugin(byte)    = "ssreflect_plugin.cmo"
     plugin(native)  = "ssreflect_plugin.cmxs"
   )
+
+  package "ltac2" (
+
+    description = "Coq LTAC2 Plugin"
+    version     = "8.12"
+
+    requires    = "coq.plugins.ltac"
+    directory   = "../user-contrib/Ltac2"
```

Comment:
I guess this is the hack you mentioned. I am afraid I haven't really followed how Ltac2 turns out to be installed.

Reply:
That's a pretty small hack, IMO not a big deal and will go away if Coq is build with Dune [meta file is generated correctly]